src/store/undeployedSafeSlice.ts

```diff
@@ -0,0 +1,56 @@
+import { type RootState } from '@/store/index'
+import { createSelector, createSlice, type PayloadAction } from '@reduxjs/toolkit'
+import type { PredictedSafeProps } from '@safe-global/protocol-kit/dist/src/types'
+
+type UndeployedSafeSlice = { [address: string]: PredictedSafeProps }
+
+type UndeployedSafeState = { [chainId: string]: UndeployedSafeSlice }
+
+const initialState: UndeployedSafeState = {}
+
+export const undeployedSafeSlice = createSlice({
+  name: 'undeployedSafe',
+  initialState,
+  reducers: {
+    setUndeployedSafe: (_, action: PayloadAction<UndeployedSafeState>): UndeployedSafeState => {
```

Comment:
This sets the entire state (i.e. all chains), not just one Safe, doesn't it?

Reply:
Yeah this is a leftover of the initial implementation of the slice. Its not used anywhere so I would remove it for now.